tests/test_main.py

```diff
@@ -67,51 +14,17 @@ def test_create_awesome_web_page_method(engineer, printed_message):
     f = io.StringIO()
     with redirect_stdout(f):
         assert engineer.create_awesome_web_page() == "<h1>Hello world</h1>"
```

Comment:
The `test_create_awesome_web_page_method` function does not include a test case for the `FullStackDeveloper` class. Since `FullStackDeveloper` inherits from both `FrontendDeveloper` and `BackendDeveloper`, it might be beneficial to ensure that the `create_awesome_web_page` method works as expected for `FullStackDeveloper` as well.